frontend/dockerfile/builder/build.go

```diff
@@ -59,6 +61,9 @@ func Build(ctx context.Context, c client.Client) (_ *client.Result, err error) {
 			}
 			return res, err
 		} else if ref, cmdline, loc, ok := parser.DetectSyntax(src.Data); ok {
+			if runtime.GOOS == "windows" {
+				ctx = context.WithValue(ctx, appdefaults.ContextKeyCustomFrontend, true)
```

Comment:
How would this work in remote frontend. I think this would need to be in gateway frontend pkg.